instock_connector/config/myinstock.example.yaml

```diff
@@ -13,3 +13,7 @@ instock-asrs-1:
     instock_site_id: x.y.z-A
     # Position of the ASRS in map space
     pose: { "x": 0, "y": 0, "yaw": 0 }
+    # Map of WMS product IDs to InStock article IDs.
+    wms_translation_table:
+      - R8S1B1-KraftParmesan: R8S1B1  # WMS_CHEESE: INSTOCK_CHEESE
+      - R7S1B1-BarillaPasta: R7S1B1  # WMS_PASTA: INSTOCK_PASTA

```

Comment:
I would add this to the internal confg files we have on GitLab and maybe mat this less "robot space" specific in the public repo.

Reply:
+1 to make it a config file with sample products that need mapping